index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require('functions.php');
+
+$allData = retrieveData();
+
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Collection</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <div class="nav-panel">
+        <nav>
+            <div class="link underline"><p>Collection</p></div>
```

Comment:
Using the link class for something that's not a link, especially right next to a genuine use of a link class isn't very semantic